src/components/UserSelector.tsx

```diff
@@ -73,10 +61,10 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(authorActions.setAuthor(user));
```

Comment:
This line sets the author when a user is clicked. Ensure that the `setAuthor` action is correctly implemented in the `authorActions` to update the state as expected.